modules/default/weather/hourly.njk

```diff
@@ -21,15 +21,27 @@
                         {% endif %}
                     </td>
                 {% endif %}
+                {% if config.showHumidity %}
+                  <td class="align-left bright humidity-hourly">
+                          {{ hour.humidity }}
+                          <i class="wi wi-humidity humidity-icon"></i>
+                  </td>
+                {% endif %}
                 {% if config.showPrecipitationAmount %}
-                    <td class="align-right bright precipitation-amount">
-                        {{ hour.precipitationAmount | unit("precip", hour.precipitationUnits) }}
-                    </td>
+                  {% if config.hideZeroes and hour.precipitationAmount==0 %}
```

Comment:
this empty if seems hard to read.
would prefer a if (!config.hideZeroes or hour.precipitationAmount > 0)